### ISSUE TITLE
Missing label

### ISSUE THREAD

**Issue by brianteeman** [j2commerce/j2commerce#612]
instead of `<small><strong>` this should be a label for the input field. All inputs must have an associated label for accessibility

https://github.com/j2commerce/j2commerce/blob/4d214696bc7403c46e003f778cd8bdabe3602e5f/administrator/components/com_j2commerce/tmpl/product/form_ajax_avfilter.php#L73-L77

**Comment by newlinewebdesign** [comment]
Fix submitted in PR #624
